apps/explorer-e2e/src/fixtures/constants.ts

```diff
@@ -0,0 +1,92 @@
+// General/Shared
+export const SEARCHBAR = `input[name="query"]`
+export const APP_NAME = 'explorer-identity-appName'
+export const APP_NETWORK = 'explorer-identity-network'
+export const DATUM_VALUE = 'explorer-datum-value'
+
+// Home page
+export const METRICS_ITEM = 'explorer-metrics-item'
+export const LATEST_BLOCKS_ITEM = 'explorer-latestBlocks-item'
+export const TOP_HOSTS_ITEMS = 'explorer-topHosts-item'
+
+// Contract page
```

Comment:
I would lean towards colocating these strings with actual fixtures / helpers they are used in rather than the extra indirection of a generic constants file, for example SEARCHBAR is only used in ExplorerApp it could at least be in that file or inlined. The renew button strings could be in a contracts fixture file or inlined there in methods like `clickRenewedFrom`.

No need to change now, just some thoughts to consider in future iteration.